transformer_engine/common/__init__.py

```diff
@@ -123,7 +151,25 @@ def _load_nvrtc():
     return ctypes.CDLL(f"libnvrtc.{_get_sys_extension()}", mode=ctypes.RTLD_GLOBAL)
 
 
+def _load_cublas():
```

Comment:
Is this redundant with `_load_nvidia_cuda_library`?

Reply:
Oops, from an earlier iteration, forgot to delete.